framework/auth/core.py

```diff
@@ -1292,7 +1297,7 @@ def merge_user(self, user):
                 node.save()
 
         # - projects where the user was the creator
-        for node in user.node__created:
+        for node in user.created:
```

Comment:
As discussed in person: so this doesn't actually appear to be *used* anywhere. Can it be removed entirely?